modules/bmo-patching-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -0,0 +1,58 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="bmo-patching-the-hostfirmwarecomponents-resource_{context}"]
+= Patching the HostFirmwareComponents resource
+
+You can patch the `HostFirmwareComponents` resource for a provisioned host, cordon and drain the host, annotate the host to reboot, and then uncordon the host to make the changes take effect.
+
+.Prerequisites
+
+* The `HostUpdatePolicy` must have `firmwareUpdates` set to `onReboot`.
+
+.Procedure
+
+. Patch the `HostFirmwareComponents` resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch hostfirmwarecomponents <hostname> --type merge -p \ <1>
+    '{"spec": {"updates": [{"component": "<type>", \ <2>
+                        "url": "<url>"}]}}' <3>
+----
+<1> Replace `<hostname>` with the name of the host.
+<2> Replace `<type>` with the type of component. Specify `bios` or `bmc`.
+<3> Replace `<url>` with the URL for the component.
+
+. Cordon the host by running the following command:
++
+[source,terminal]
+----
+$ oc cordon <hostname>
+----
++
+The BMO updates the `operationalStatus` to `servicing`.
```

Comment:
Nit: It might be helpful to add an `.Example output` from the terminal here.

Reply:
It would only provide "node/<hostname> cordoned", which doesn't really add a lot of value.